pkg/util/storage.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
I probably wasn't clear about pkg/util/storage being the package, sorry.

On Nov 18, 2015, at 3:02 PM, Brendan Burns <notifications@github.com> wrote:

In pkg/util/storage.go
<https://github.com/kubernetes/kubernetes/pull/16009#discussion_r45252335>:

> @@ -0,0 +1,25 @@
> +/*
> +Copyright 2015 The Kubernetes Authors All rights reserved.

@smarterclayton <https://github.com/smarterclayton> you forced me to push
this out of cloud provider like 2 weeks ago and you agreed util.Storage was
ok
<https://github.com/kubernetes/kubernetes/pull/16009#issuecomment-150346159>

I move it to pkg/util/storage/ I'd really rather not move it again.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/16009/files#r45252335>.